checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -131,6 +131,8 @@ def _check_suppression(self, record: Record, suppression: dict[str, Any]) -> boo
             return any(self.bc_integration.repo_matches(account) for account in suppression['accountIds'])
         elif type == 'Resources':
             for resource in suppression['resources']:
+                if 'accountId' not in resource:
+                    return False
                 if self.bc_integration.repo_matches(resource['accountId']) \
                         and resource['resourceId'] == f'{record.repo_file_path}:{record.resource}':
                     return True
```

Comment:
resource suppression type is not part of CVEs scope

Reply:
@Saarett right, but why not add it here as well in order to prevent such errors ?

Reply:
It depends on the suppression rule of resources which is not in our domain and we don't really know